src/content/manifest.json

```diff
@@ -1 +1,488 @@
-[]
+[
```

Comment:
@alexissmirnov I think that this file is not suggested to be commited

Reply:
we should gitignore it then